source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -594,6 +596,225 @@ For example: If there are 10 available pieces of Medium size, 15 of Large and 5
 <quantity>30</quantity>
 ~~~
 
+
+### Size Variation Attributes
+
+Size variations is an enhancement to the current XML feed, that introduces new nodes (children)
+that include the information of size variations nested to the parent product.
+The parent product should be exactly the same with the current XML specification (including
```

Comment:
Maybe state that "For the time being it should be exactly the same", in order to show our intent (that we need it until variations are fully adopted) because it does not really make sense to ask for the same information twice.